roles/idrac_user/README.md

```diff
@@ -0,0 +1,347 @@
+  </tr>
+  </tr>
+  <tr>
+    <td>custom_privilege</td>
+    <td>false</td>
+    <td></td>
+    <td></td>
+    <td>int</td>
+    <td>- Provide the custom role-based authority privileges allowed for the user.</br>- To create a custom privilege, add up the privilege decimal values as defined below.</br>Login - 1</br>Configure - 2</br>Configure Users - 4</br>Logs - 8</br>System Control - 16</br>Access Virtual Console - 32</br>Access Virtual Media - 64</br>System Operations - 128</br>Debug - 256</br>- The value has to be in the range 0-511.</td>
+  </tr>
+  <tr>
+    <td>ipmi_lan_privilege</td>
+    <td>false</td>
+    <td></td>
+    <td>["Administrator","ReadOnly","Operator","No Access"]</td>
+    <td>str</td>
+    <td>- The Intelligent Platform Management Interface LAN privilege level assigned to the user.</td>
+  </tr>
+  <tr>
+    <td>ipmi_serial_privilege</td>
+    <td>false</td>
+    <td></td>
+    <td>["Administrator","ReadOnly","Operator","No Access"]</td>
+    <td>str</td>
+    <td>- The Intelligent Platform Management Interface Serial Port privilege level assigned to the user.</br>- This option is only applicable for rack and tower servers.</td>
+  </tr>
+  <tr>
+    <td>enable</td>
+    <td>false</td>
+    <td></td>
+    <td></td>
+    <td>bool</td>
+    <td>Provide the option to enable or disable a user from logging in to iDRAC.</td>
+  </tr>
+   <tr>
+    <td>sol_enable</td>
+    <td>false</td>
+    <td></td>
+    <td></td>
+    <td>bool</td>
+    <td>Enables Serial Over Lan (SOL) for an iDRAC user.</td>
+  </tr>
+   <tr>
+    <td>protocol_enable</td>
+    <td>false</td>
+    <td></td>
+    <td></td>
+    <td>bool</td>
+    <td>Enables SNMPv3 protocol for the iDRAC user.</td>
+  </tr>
+  <tr>
+    <td>authentication_protocol</td>
+    <td>false</td>
+    <td></td>
+    <td>["None","SHA5","MD5"]</td>
+    <td>str</td>
+    <td>- This option allows to configure one of the following authentication protocol types to authenticate the iDRAC user.</br>- Secure Hash Algorithm C(SHA).</br>- Message Digest 5 C(MD5).</br>- If C(None) is selected, then the authentication protocol is not configured.</td>
+  </tr>
+  <tr>
+    <td>privacy_protocol</td>
+    <td>false</td>
+    <td></td>
+    <td>["None","DES","AES"]</td>
+    <td>str</td>
+    <td>- This option allows to configure one of the following privacy encryption protocols for the iDRAC user.</br>- Data Encryption Standard C(DES).</br>- Advanced Encryption Standard C(AES).</br>- If C(None) is selected, then the privacy protocol is not configured.</td>
+  </tr>
+
+  
+## Fact variables
+
+<table>
+<thead>
+  <tr>
+    <th>Name</th>
+    <th>Sample</th>
+    <th>Description</th>
+  </tr>
+</thead>
+  <tbody>
+    <tr>
+      <td>idrac_user_out</td>
```

Comment:
Please update proper return msg, which is coming from modules, idrac_user_out will return module output, I can see status and error_info is missing

https://github.com/dell/dellemc-openmanage-ansible-modules/blob/collections/plugins/modules/idrac_user.py#L159

Reply:
resolving it as status doesn't add value